models/meshery_pattern_persister.go

```diff
@@ -62,6 +63,17 @@ func (mpp *MesheryPatternPersister) DeleteMesheryPattern(id uuid.UUID) ([]byte,
 	return marshalMesheryPattern(&pattern), nil
 }
 
+// DeleteMesheryPatterns takes in a array of profile ids and delete those if exist
+func (mpp *MesheryPatternPersister) DeleteMesheryPatterns(/*ids []uuid.UUID*/) ([]byte, error) {
+	// for _, id := range(ids) {
+	// 	pattern := MesheryPattern{ID: &id}
+	// 	mpp.DB.Delete(&pattern)
+	// }
+	logrus.Info("Came here did nothing-.................")
+
+	return nil, nil // ...
+}
+
```

Comment:
What is pattern-persistor, I couldn't reach to this code-block while making normal requests, but was throwing error if there is no such function signature. Is is cache or something related? How could I reach the code-block?

Reply:
This is where it is defined https://github.com/meshery/meshery/pull/4324/files/84e0fed1f908a03c41e4daf11fbaf005ab0ec40d#diff-b008d466a53041b957f27ef5d4cf0a6cb5a7b35860fe6511555d67ea3154cc21R14